pkgs/development/python-modules/python-poppler/default.nix

```
@@ -0,0 +1,65 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  pytestCheckHook,
+  setuptools,
+  meson-python,
+  ninja,
+  meson,
+  poppler,
+  pkg-config,
+  pybind11,
+}:
+
+buildPythonPackage rec {
+  pname = "python-poppler";
+  version = "0.4.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "python_poppler";
+    hash = "sha256-5spcI+wCNQvyzvhaa/nxsmF5ZDbbR4F2+dJPsU7uzGo=";
+  };
+
+  patches = [
+    # Prevent Meson from downloading pybind11, use system version instead
+    ./use_system_pybind11.patch
+  ];
+
+  build-system = [ setuptools ];
+
+  build-inputs = [ pybind11 ];
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+  ];
+
+  dependencies = [
+    meson-python
+    poppler
+  ];
+
+  preBuild = ''
+    # Tell pkg-config where to find pybind11
+    export PKG_CONFIG_PATH=${pybind11}/share/pkgconfig''${PKG_CONFIG_PATH:+:}$PKG_CONFIG_PATH
+  '';
+
+  checkInputs = [ pytestCheckHook ];
+
+  pythonImportCheck = [ "poppler" ];
+
+  meta = {
+    description = "Python binding to poppler-cpp";
+    homepage = "https://github.com/cbrunet/python-poppler";
+    changelog = "https://cbrunet.net/python-poppler/changelog.html";
+    license = lib.licenses.gpl2Only;
```

According to the file headers (e.g., https://github.com/cbrunet/python-poppler/blob/master/src/poppler/__init__.py) it looks like `GPLv2+`.

```suggestion
    license = lib.licenses.gpl2Plus;
```

---

Thank you, fixed it!